morebits.js

```diff
@@ -730,14 +730,17 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 			var moreButton = more[1];
 
 			var sublist = {
-				type: '_dyninput_element',
-				label: data.sublabel || data.label,
-				name: data.name,
-				value: data.value,
-				size: data.size,
+				type: '_dyninput_row',
 				remove: false,
 				maxlength: data.maxlength,
-				event: data.event
+				event: data.event,
+				inputs: data.inputs || [{
+					// compatibility
```

Comment:
For my understanding, what's going on with the compatibility sub-section here?

Reply:
So other scripts using `dyninput` will continue to work as they did. We don't have to do that for `_dyninput_element`, which is private.